src/platform/user/authentication/utilities.js

```diff
@@ -126,13 +117,9 @@ export function verify(version = 'v0') {
   return redirect(sessionTypeUrl('verify', version), 'verify-link-clicked');
 }
 
-export function logout(version = 'v0') {
+export function logout(version = 'v0', clickedEvent = 'logout-link-clicked') {
```

Comment:
It doesn't look like this new parameter is tested for, right?

Reply:
ill add a couple of tests